Papr/Scenes/Login/LoginViewModel.swift

```diff
@@ -81,6 +82,14 @@ class LoginViewModel: LoginViewModelInput, LoginViewModelOuput, LoginViewModelTy
         }
     }()
     
+    // MARK: Action
+    lazy var closeAction: CocoaAction = {
+        return CocoaAction { [unowned self] _ in
+//            self.authenticate()
```

Comment:
We can remove unnecessary comments.

Reply:
Why not returning just this: `self.sceneCoordinator.pop(animated: true)`? 
`sceneCoordinator.pop(animated: ...)` is an `Observable<Void>`